app/renderer/components/navigation/navigationBar.js

```diff
@@ -180,12 +263,13 @@ class NavigationBar extends React.Component {
       return null
     }
 
-    return <div id='navigator'
+    return <div id='navigationBar'
       ref='navigator'
+      data-test-id='navigationBar'
       data-frame-key={this.props.activeFrameKey}
       className={cx({
         titleMode: this.props.titleMode,
-        [css(styles.navigator_wide)]: this.props.isWideURLbarEnabled
+        [css(styles.navigationBar, (this.props.isDarwin && this.props.isFullScreen) && styles.navigationBar_isDarwin_isFullScreen, this.props.titleMode && styles.navigationBar_titleMode, this.props.isWideURLbarEnabled && styles.navigationBar_wide)]: true
```

Comment:
Yes, it includes not only media queries but also wildcards with pseudo parent (which is `:not(.titleMode)`) so I think this needs some additional work.

The media query settings seem to need revisiting. The wildcard setting would be replaced by creating an alternative common style such as `style.animation` and apply them to all of the child elements. I think this is the better option than anything else. What do you think?